tests/Bridge/Doctrine/Transport/DoctrineTransportTest.php

```diff
@@ -368,7 +369,7 @@ public function testTransportCannotCreateAnExistingTask(): void
         $serializer = $this->createMock(SerializerInterface::class);
 
         $task = $this->createMock(TaskInterface::class);
-        $task->expects(self::once())->method('getName')->willReturn('foo');
+        $task->expects(self::exactly(2))->method('getName')->willReturn('foo');
```

Comment:
As we already perform assertions later, maybe we can use `$task = new NullTask('foo');`?